lib/reboot_cmds.py

```diff
@@ -48,7 +48,8 @@ def fix_pro_pkg_holds(cfg: config.UAConfig):
                 return
 
     LOG.info("Attempting to remove Ubuntu Pro FIPS package holds")
-    fips = FIPSEntitlement(cfg)
+    fips = entitlement_factory(cfg=cfg, name="fips")
```

Comment:
That's fair, I was trying to avoid  calling this in the `__init__` method itself, but I think you are right that this is a better approach. Let me update it